src/internal/k8s/tunnel.go

```diff
@@ -88,6 +88,33 @@ func PrintConnectTable() error {
 	return nil
 }
 
+// NewTunnelFromServiceURL takes a serviceURL and parses it to create a tunnel to the cluster. The string is expected to follow the following format:
+// Example serviceURL: http://{SERVICE_NAME}.{NAMESPACE}.svc.cluster.local:{PORT}
+func NewTunnelFromServiceURL(serviceURL string) (*Tunnel, error) {
+
+	// Remove the protocol from the URL
+	serviceURL = strings.TrimPrefix(serviceURL, "http://")
```

Comment:
Think we'd be better served using a URL parsing library than manually hacking this:  https://golangbyexample.com/parse-url-golang/